Lecture2/Task1/builder/Guitar.java

```diff
@@ -0,0 +1,52 @@
+package builder;
+
+import product.Brand;
+
+public class Guitar {
```

Comment:
you have to create hierarchy of guitar

Create to class acustic and electric guitar which extents from guitar

Reply:
done